api/egl.xml

```diff
@@ -1025,6 +1025,9 @@
     <enums namespace="EGL" start="0x34A0" end="0x34AF" vendor="ANGLE" comment="Reserved for Ken Russell - ANGLE (via github pull request)">
             <unused start="0x34A0" end="0x34AF"/>
     </enums>
+    <enums namespace="EGL" start="0x34D0" end="0x34DF" vendor="ANGLE" comment="Reserved for Peng Huang - ANGLE (via github pull request)">
```

Comment:
Could you move this down to line 1037/1040 so it remains in numerical order? Thanks.

Reply:
Done

Reply:
LGTM